packages/react-components/theme-designer/src/components/ColorTokens/AccessibilityList.tsx

```diff
@@ -24,33 +26,54 @@ export interface AccessibilityListProps {
 export const AccessibilityList: React.FunctionComponent<AccessibilityListProps> = props => {
   const { brand, brandColors, colorOverride, onNewOverride, theme } = props;
 
-  const { pass, fail } = getAccessibilityChecker(theme);
-  const failList = Object.keys(fail);
+  const { all, fail } = getAccessibilityChecker(theme);
+
+  const failContrastNum = () => {
+    if (Object.keys(fail).length === 0) {
+      return (
+        <Badge appearance="outline" color="important">
+          <CheckmarkCircleRegular color="green" /> &nbsp; All contrast requirements met &nbsp;
+        </Badge>
+      );
+    } else if (Object.keys(fail).length === 1) {
+      return (
+        <Badge appearance="outline" color="important">
+          <WarningRegular color="red" /> &nbsp; 1 token miss required contrast &nbsp;
+        </Badge>
+      );
+    } else {
+      return (
+        <Badge appearance="outline" color="important">
+          <WarningRegular color="red" /> &nbsp; {Object.keys(fail).length} tokens miss required contrast &nbsp;
+        </Badge>
+      );
+    }
+  };
```

Comment:
seeing as everything is the same except for the text, these two can be combined and a ternary used for the text output